CPAC/utils/utils.py

```diff
@@ -2657,3 +2657,192 @@ def flip_orientation_code(code):
     """Reverts an orientation code by flipping R↔L, A↔P, and I↔S."""
     flip_dict = {"R": "L", "L": "R", "A": "P", "P": "A", "I": "S", "S": "I"}
     return "".join(flip_dict[c] for c in code)
+
+
+def get_fmap_type(metadata):
+    """Determine the type of field map from metadata.
+
+    reference: https://bids-specification.readthedocs.io/en/latest/modality-specific-files/magnetic-resonance-imaging-data.html#case-1-phase-difference-map-and-at-least-one-magnitude-image
```

Comment:
Let's make this a permalink:

```suggestion
    reference: https://bids-specification.readthedocs.io/en/v1.10.0/modality-specific-files/magnetic-resonance-imaging-data.html#case-1-phase-difference-map-and-at-least-one-magnitude-image
```